openedx/core/djangoapps/catalog/utils.py

```diff
@@ -230,15 +231,21 @@ def get_currency_data():
         return []
 
 
-def format_price(price, symbol='$', code='USD'):
+def format_price(price, symbol, code):
     """
     Format the price to have the appropriate currency and digits..
 
     :param price: The price amount.
-    :param symbol: The symbol for the price (default: $)
-    :param code: The currency code to be appended to the price (default: USD)
+    :param symbol: The symbol for the price (default: configured currency symbol)
+    :param code: The currency code to be appended to the price (default: configured currency)
     :return: A formatted price string, i.e. '$10 USD', '$10.52 USD'.
     """
+    if not code:
+        code = settings.PAID_COURSE_REGISTRATION_CURRENCY[0]
+
+    if not symbol:
+        symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
```

Comment:
Since we have removed the default parameters, are these checks actually required? Because now users will have to call the function with all the parameters present. And if the user explicitly passes `""` or `None` as `symbol` or `code`in the function, is it okay to change that ? instead of throwing some error?
Just curious.

Reply:
done.